package.json

```diff
@@ -103,6 +103,7 @@
 		"@babel/core": "7.8.4",
 		"@babel/preset-env": "7.8.4",
 		"@babel/register": "7.7.7",
+		"@wordpress/annotations": "1.12.2",
```

Comment:
I'm not sure this is needed here? Importing the package in blocks should be enough for it to get added as a dependency when the block bundle is loaded.

Reply:
That's what I thought, too, but then [the Jest tests](https://travis-ci.com/github/Automattic/jetpack/jobs/392949639) were failing without it.